### PR TITLE
Update commons packages, use inspect container env to find DATA_FOLDER

### PR DESCRIPTION
**Pull request by cedmail**

### CODE REVIEW

File: maven-jahia-plugin/src/main/java/org/jahia/utils/maven/plugin/DeployMojo.java [hunk 1]
```diff
@@ -193,7 +193,14 @@ public void doExecute() throws MojoExecutionException, MojoFailureException {
                     boolean isStandardModule = project.getGroupId().equals("org.jahia.module") || project.getGroupId().endsWith(".jahia.modules");
                     String filename = project.getArtifactId() + "-" + project.getVersion() + "." + "jar";
                     File source = new File(output, filename);
-                    String dockerDatadir = "/data/digital-factory-data";
+                    String dockerDatadir = "/var/jahia";
+                    String[] env = dockerClient.inspectContainerCmd(targetContainerName).exec().getConfig().getEnv();
+                    for (String s : env) {
+                        if(s.startsWith("DATA_FOLDER")) {
```
Comment (tdraier) [inline maven-jahia-plugin/src/main/java/org/jahia/utils/maven/plugin/DeployMojo.java:199]:
```suggestion
                        if(s.startsWith("DATA_FOLDER=")) {
```

File: maven-jahia-plugin/src/main/java/org/jahia/utils/maven/plugin/osgi/SourceFolderMojo.java [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.jar.Attributes;
+import java.util.jar.JarFile;
+import java.util.jar.Manifest;
+
+/**
+ * A goal that checks the dependencies of a generated OSGi bundle JAR against the project dependencies, and reports
```
Comment (tdraier) [inline maven-jahia-plugin/src/main/java/org/jahia/utils/maven/plugin/osgi/SourceFolderMojo.java:24]: the description does not look to match this class :-)

File: maven-jahia-plugin/src/main/java/org/jahia/utils/maven/plugin/osgi/SourceFolderMojo.java [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+        }
+        try (JarFile jarFile = new JarFile(artifactFile)) {
+            Manifest manifest = jarFile.getManifest();
+            if (manifest.getMainAttributes() == null) {
+                throw new MojoExecutionException("Error reading OSGi bundle manifest data from artifact " + artifactFile);
+            }
+            String groupId = project.getGroupId();
+            boolean isModule = groupId.equals("org.jahia.module") || groupId.endsWith(".jahia.modules") || manifest.getMainAttributes().containsKey(new Attributes.Name("Jahia-Module-Type"));
+            if (isModule) {
+                String sourceFolder = "/data/digital-factory-data/sources/" + project.getBasedir().getName();
```
Comment (tdraier) [inline maven-jahia-plugin/src/main/java/org/jahia/utils/maven/plugin/osgi/SourceFolderMojo.java:82]: shouldn't we use /var/jahia by default ?

Comment (cedmail) [reply maven-jahia-plugin/src/main/java/org/jahia/utils/maven/plugin/osgi/SourceFolderMojo.java:82]: good catch, forgot this one

Comment (cedmail) [reply maven-jahia-plugin/src/main/java/org/jahia/utils/maven/plugin/osgi/SourceFolderMojo.java:82]: in fact it is not activated and as we now try to find the sources in the data folder of Jahia during runtime if they are not found in Jahia-Source-Folders entry, we should probably get rid of this class

Comment (cedmail) [reply maven-jahia-plugin/src/main/java/org/jahia/utils/maven/plugin/osgi/SourceFolderMojo.java:82]: @tdraier

Comment (tdraier) [reply maven-jahia-plugin/src/main/java/org/jahia/utils/maven/plugin/osgi/SourceFolderMojo.java]: yes i was not clear about the usage of it, you can remove it

**Review by tdraier — APPROVED** [review]

**Pull request merged**